tcf_website/management/commands/fix_multiple_objects_returned.py

```diff
@@ -0,0 +1,113 @@
+from functools import reduce
+from operator import or_
+
+from django.core.management.base import BaseCommand
+from django.db.models import Case, Count, F, FloatField, Q, Sum, Value, When
+
+from tcf_website.models import CourseInstructorGrade
+
+
+class Command(BaseCommand):
+    """This fixes existing `MultipleObjectsReturned` errors"""
+
+    help = 'Fix `MultipleObjectsReturned` errors that were caused by inconsistent email data'
+
+    def handle(self, *args, **options):
+        self.verbosity = options['verbosity']
+
+        # Number of cases we want to fix
+        qs = CourseInstructorGrade.objects\
+            .exclude(course=None)\

```

Comment:
What kinds of CourseInstructorGrade instances don't have `course`?

Reply:
I couldn't find a pattern, and that's out of the scope of this PR, but I'm suspecting the courses were left out when we ran the `april7.sql` three times, just like [this issue](https://discord.com/channels/752270432329269328/767189787131117609/783027475084869686). Luckily, the number wasn't too big iirc.